src/deepqmc/gnn/graph.py

```diff
@@ -280,13 +280,13 @@ def GraphUpdate(
     def update_graph(graph):
         nodes, edges = graph
 
-        if update_edges_fn:
-            edges = update_edges_fn(edges)
-
         if update_nodes_fn:
             aggregated_edges = aggregate_edges_for_nodes_fn(nodes, edges)
             nodes = update_nodes_fn(nodes, aggregated_edges)
 
+        if update_edges_fn:
```

Comment:
We first update the nodes using the current edges, and only update the edges afterwards. This only makes a difference if `deep_features` is used. Things like this are much easier to do now that the input dimensions of the MLPs are inferred, since we don't have to worry about the FermiNet and DeepErwin first layer special cases.